components/tools/OmeroWeb/test/integration/test_api_login.py

```diff
@@ -179,10 +226,13 @@ def test_login_example(self):
             'server': server_id,
             # 'csrfmiddlewaretoken': token,
         }
-        login_rsp = django_client.post(login_url, data)
-        login_json = json.loads(login_rsp.content)
-        assert login_json['success']
-        event_context = login_json['eventContext']
+        login_rsp = django_client.post(
```

Comment:
Mostly comment to myself - this works because `Client.post()` encodes the passed-in `data` according to the given `content_type` for `application/json` and `MULTIPART_CONTENT`; all other `content_types` would cause `data` to be passed as is.